director/commands/celery.py

```diff
@@ -18,9 +18,9 @@ def beat(dev_mode, beat_args):
     """Start the beat instance"""
     args = [
         "celery",
-        "beat",
         "-A",
         "director._auto:cel",
+        "beat",
```

Comment:
Can you explain this change please?
Is it mandatory and introduced with Celery v5?

Reply:
see: https://github.com/ovh/celery-director/pull/137#discussion_r826368412